src/registrar/admin.py

```diff
@@ -262,9 +285,17 @@ class UserContactInline(admin.StackedInline):
     model = models.Contact
 
 
-class MyUserAdmin(BaseUserAdmin):
+class UserAdmin(BaseUserAdmin):
```

Comment:
Blocking: There is a django UserAdmin class. I think we should keep MyUserAdmin, creating a class with a name that matches one of Django's makes me nervous.